tests/e2e/pageobjects/kubernetes/DexLoginPage.ts

```diff
@@ -0,0 +1,58 @@
+/*********************************************************************
+ * Copyright (c) 2019-2023 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import 'reflect-metadata';
+import { injectable, inject } from 'inversify';
+import { CLASSES } from '../../configs/inversify.types';
+import { Logger } from '../../utils/Logger';
+import { By } from 'selenium-webdriver';
+import { TimeoutConstants } from '../../constants/TimeoutConstants';
+import { DriverHelper } from '../../utils/DriverHelper';
+
+@injectable()
+export class DexLoginPage {
```

Comment:
`e2e/pageobjects/login` folder is more suitable for login page classes like this one, IMHO

Reply:
But DexLoginPage class contains elements and base methods, and in KubernetesLoginPage - implement login method combined from those methods. Like RegularUserOcpCheLoginPage and OcpLoginPage. I think it will be nice to review all this section in some following task

Reply:
Okay, and both "RegularUserOcpCheLoginPage" and "OcpLoginPage" classes are placed into the same `e2e/pageobjects/login` folder.

It makes it simpler to navigate inside `e2e/pageobjects` folder and localize page object.
Why not to keep every login related classes in `e2e/pageobjects/login` folder?